backend/mysql_wrapper/sql_executor.py

```diff
@@ -0,0 +1,222 @@
+from datetime import date
+from backend.student.member import Member
+from backend.mysql_wrapper.sql_connection import *
+from backend.student.member_enums import *
+
+def __initializeDatabaseSchemaIfNotExists(conn: MySQLConnection = None) -> None:
+    conn = getSQLConnection() if conn is None else conn
+    cursor = conn.cursor()
+    cursor.execute("CREATE DATABASE IF NOT EXISTS celesta_data;")
+    cursor.execute("USE celesta_data;")
+
+    # Initialize all tables here
+
+    cursor.execute("""CREATE TABLE IF NOT EXISTS members (
```

Comment:
Can these strings be assigned to variables like in insertInstrument? I know they're not being reused at the moment but I think it would make it a bit cleaner overall.

Reply:
resolving so as not to block merge